issue.go

```diff
@@ -424,16 +424,30 @@ type Option struct {
 // UnmarshalJSON will transform the JIRA time into a time.Time
 // during the transformation of the JIRA JSON response
 func (t *Time) UnmarshalJSON(b []byte) error {
-	// Ignore null, like in the main JSON package.
-	if string(b) == "null" {
+	s := strings.Trim(string(b), "\"")
+
+	// Ignore null and empty strings.
+	if s == "" || s == "null" {
 		return nil
 	}
-	ti, err := time.Parse("\"2006-01-02T15:04:05.999-0700\"", string(b))
-	if err != nil {
-		return err
+
+	var ti time.Time
+	var err error
+	if strings.Contains(s, "Z") {
```

Comment:
This might be fine, but probably better to just check the last character where we'd expect it to be for this format. 
```suggestion
	if strings.HasSuffix(s, "Z") {
```

Reply:
Yeah, I'd do the last char. That's where it will always be.